Oxide.Core/Libraries/Permission.cs

```diff
@@ -124,15 +125,24 @@ public void Export(string prefix = "auth")
             Interface.Oxide.DataFileSystem.WriteObject(prefix + ".users", userdata);
         }
 
+        /// <summary>
+        /// Saves both user/group data
+        /// </summary>
+        public void SavePermissionData()
+        {
+            SaveUsers();
+            SaveGroups();
+        }
+
         /// <summary>
         /// Saves all permissions data to the datafile
         /// </summary>
-        private void SaveUsers() => ProtoStorage.Save(userdata, "oxide.users");
+        public void SaveUsers() => ProtoStorage.Save(userdata, "oxide.users");
```

Comment:
These being public poses an issue whereas plugins could trigger a save and corrupt data.